aquasec-server/templates/rbac.yaml

```diff
@@ -0,0 +1,42 @@
+{{- if .Values.rbac.enabled -}}
+{{- if not .Values.rbac.roleRef }}
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Release.Name }}-cluster-role
+  labels:
+    app: {{ .Release.Name }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+# TODO: Make role's permissions much more specific for server component needs
```

Comment:
@oranmoshai please help with rbac roles here too

Reply:
Currently there is no need for rbac in this section.